code/modules/mob/living/simple_animal/friendly/farm_animals.dm

```diff
@@ -177,6 +179,9 @@
 	else
 		..()
 
+/mob/living/simple_animal/cow/proc/reset_canmove()

```

Comment:
why did you make a proc that does nothing but call another proc am I being trolled

Reply:
yes